tests/foreman/api/test_contentview.py

```diff
@@ -896,28 +777,22 @@ def test_promote_composite_cv_multiple_2(self):
         """
         composite_cv = entities.ContentView(
             composite=True,
-            organization=self.org.id,
+            organization=self.org,
         ).create()
         self.add_content_views_to_composite(composite_cv, random.randint(3, 5))
         composite_cv.publish()
+        composite_cv = composite_cv.read()
 
-        cvv = entities.ContentViewVersion(
```

Comment:
After all the changes, I am not sure whether we need ```ContentViewVersion``` entity at all. Do we need it?